From ee12fd4fd9b7fb77876479e89d4b37e360f171d2 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 30 Apr 2015 11:43:13 +0800
Subject: [PATCH 0107/5242] MLK-10783-5 usb: phy: mxs: add "fsl,imx6ul-usbphy"
 compatible string

commit  91f3952d8c13e9792030ce2a7754854909eedaf0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add "fsl,imx6ul-usbphy" compatible string

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit dba30c6e41a9380b41715eca54792bfaa64693d0)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/phy/phy-mxs-usb.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/phy/phy-mxs-usb.c b/drivers/usb/phy/phy-mxs-usb.c
index 26b6816..a4dc53c 100644
--- a/drivers/usb/phy/phy-mxs-usb.c
+++ b/drivers/usb/phy/phy-mxs-usb.c
@@ -182,6 +182,7 @@ struct mxs_phy_data {
 };
 
 static const struct of_device_id mxs_phy_dt_ids[] = {
+	{ .compatible = "fsl,imx6ul-usbphy", .data = &imx6sx_phy_data, },
 	{ .compatible = "fsl,imx6sx-usbphy", .data = &imx6sx_phy_data, },
 	{ .compatible = "fsl,imx6sl-usbphy", .data = &imx6sl_phy_data, },
 	{ .compatible = "fsl,imx6q-usbphy", .data = &imx6q_phy_data, },
-- 
1.7.9.5

